.github/workflows/release.yml

```diff
@@ -9,44 +9,37 @@ jobs:
     name: releaser
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout
+      - name: Checkout
```

Comment:
I think you made typos.
Should be grouped together (with the new line right before the `-`) :
```yaml
- name: xxx
  uses: xxx

- name: yyy
  run: yyy
```

Reply:
I don't think so